tombstone_gc.cc

```diff
@@ -147,7 +148,13 @@ tombstone_gc_state::get_gc_before_for_range_result tombstone_gc_state::get_gc_be
             }
             min_gc_before = check_min(s, saturating_subtract(min_repair_timestamp, propagation_delay));
             max_gc_before = check_min(s, saturating_subtract(max_repair_timestamp, propagation_delay));
-        };
+        } else if (_table_replication_factor == 1) {
```

Comment:
I think we should return immediate mode no matter if the repair history is present or not. Because as soon as the RF is 1, the repair history should not matter anymore.

Reply:
ok, makes sense

Reply:
Done in v3.